src/commands/rename.rs

```diff
@@ -92,12 +92,13 @@ where
                 .collect::<Vec<_>>();
 
             for k in keys {
-                if let Item::Table(root) = t.entry(&k) {
+                if let Some(Item::Table(root)) = table.get_mut(&k) {
```

Comment:
This is required, since `edit_each_dep` expects a `&mut Table`, where we pass `root`.

Note that this (as all other occurences is caused by the migration from `fn entry` in `0.2`, which returns a `&mut T` to `fn get_mut` which does the same. https://docs.rs/toml_edit/0.2.0/toml_edit/struct.Table.html#method.entry